data/notices.json

```diff
@@ -555,6 +555,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(lambda-nodejs): breaking change in esbuild causes import module error",
+      "issueNumber": 30717,
+      "overview": "Esbuild introduced a breaking change. This caused the import module error in lamda-nodejs module.",
+      "components": [
+        {
+          "name": "aws-cdk-lib.lambda-nodejs",
+          "version": "2.147.0 - 2.147.2"
```

Comment:
This will actually be a much wider version set than this. It will impact everyone who uses this since we set the eslint version to `0`

Reply:
good point. I see that feature was introduced in v1.175.0 https://github.com/aws/aws-cdk/commit/7a82850d8bec45f18791e269e988c5261d5238d4

Changed the version scope.